src/flows/pipes/Notification/Measurement.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {FC, useCallback, useContext} from 'react'
+import {
+  ComponentSize,
+  FlexBox,
+  AlignItems,
+  TextBlock,
+  Dropdown,
+  FlexDirection,
+  ComponentStatus,
+} from '@influxdata/clockface'
+
+import {PipeContext} from 'src/flows/context/pipe'
+import 'src/flows/pipes/Notification/Threshold.scss'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+
+interface Props {
+  readOnly?: boolean
+}
+
+const Measurement: FC<Props> = ({readOnly}) => {
+  const {data, update, results} = useContext(PipeContext)
+  const {measurement} = data
+
+  const measurements = Array.from(
```

Comment:
not for this pr, as it extends to the edge case, but it'd be nice if this component didn't render when `measurements.length === 1`, just to reduce the number of choices